File: Python/Problem1.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        allnums = nums1 + nums2
        allnums.sort()
        index = round(len(allnums)/2)-1
        median = allnums[index]
        if len(allnums) % 2 == 0 :
            median = (median + allnums[index+1]) / 2
        
        return median
        

File: TypeScript/Problem1.ts
function findMedianSortedArrays(nums1: number[], nums2: number[]): number {
    let allNums = [...nums1, ...nums2]

    allNums.sort();

    const index = Math.round(allNums.length/2)-1;
    let median = allNums[index];
    
    if(allNums.length % 2 == 0){
        median = (median + allNums[index+1])/2;
    }

    return median;
};